homeassistant/components/flux_led/select.py

```diff
@@ -202,3 +210,28 @@ async def async_select_option(self, option: str) -> None:
         """Change the remote config setting."""
         remote_config: RemoteConfig = self._name_to_state[option]
         await self._device.async_config_remotes(remote_config)
+
+
+class FluxWhiteChannelSelect(FluxConfigSelect):
+    """Representation of Flux white channel."""
+
+    _attr_options = [_human_readable_option(option.name) for option in WhiteChannelType]
+
+    @property
+    def current_option(self) -> str | None:
+        """Return the current white channel type."""
+        return _human_readable_option(
+            self.coordinator.entry.data.get(
+                CONF_WHITE_CHANNEL_TYPE, DEFAULT_WHITE_CHANNEL_TYPE.name
+            )
+        )
+
+    async def async_select_option(self, option: str) -> None:
+        """Change the white channel type."""
```

Comment:
Maybe a custom device class to make this translatable?

Reply:
Looks like its marketed as Warm & Natural elsewhere as well so probably not worth it